labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,20 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "ejp mysljylc kd kxveddknmc re jsicpdrysi" +
+      "rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd" +
+      "de kr kd eoya kw aej tysr re ujdr lkgc jv" +
+      "aoz"
+    val output = "our language is impossible to understand" +
+     "there are twenty six factorial possibilities" +
+     "so it is okay if you want to just give up" +
+     "yeq"
+     
+    val mapping = input.zip(output).toMap
+    
+    lines map {
+      _.map(mapping(_))
+    }
```

Comment:
A different name than `mapping` might also help :)

That said, generator/comprehension syntax might be the cleanest of all in this kind of situation:
```
for {L <- lines} yield {L map (mapping(_))}
```
or something like that.